frontend/src/metabase/browse/models/BrowseModels.tsx

```diff
@@ -88,9 +88,11 @@ export const BrowseModels = () => {
           <Stack mb="lg" gap="md" w="100%">
             {isEmpty ? (
               <Stack gap="lg" align="center" data-testid="empty-state">
-                <Box maw="45rem" w="100%">
-                  <ModelsVideo autoplay={0} />
-                </Box>
+                {showMetabaseLinks && (
```

Comment:
> "Show Metabase links" setting should apply to all Metabase content, not just the literal documentation links.

We should rename this. How about `showMetabaseDocs` or `showMetabaseContent`?

We can do it in a follow-up PR since this one is a [blocker for v54](https://metaboat.slack.com/archives/C864UT5CZ/p1743491930527959?thread_ts=1743491209.267609&cid=C864UT5CZ).